jenkinsfile-runner-base-image/packager-config.yml

```diff
@@ -18,15 +18,15 @@ buildSettings:
         git: https://github.com/jenkinsci/jenkinsfile-runner.git
         branch: "1.0-beta-12"
     docker:
-      base: "jenkins/jenkins:2.222.4-alpine"
+      base: "jenkins/jenkins:2.241-alpine"
       tag: "jenkinsfile-runner-base-local"
       build: true
 
 war:
   groupId: "org.jenkins-ci.main"
   artifactId: "jenkins-war"
   source:
-    version: "2.222.4"
+    version: "2.241"
```

Comment:
do we really want to use a weekly and not an LTS?

Reply:
This reduced the vulnerabilities only by 1. So I would also vote for staying on LTS.

Reply:
Switched to LTS again. They released a new version since yesterday 🙂